tools/release-note.md

```diff
@@ -14,19 +15,13 @@ $ docker pull "grafana/loki:${CIRCLE_TAG}"
 $ docker pull "grafana/promtail:${CIRCLE_TAG}"
 ```
 
-#### Binary:
-
-Choose from the assets below for the application and architecture matching your system.
-
-Example for `Loki` on the `linux` operating system and `amd64` architecture:
+#### Binary
+We provide pre-compiled binary executables for the most common operating systems and architectures.
+Pick one from the list below and download it:
 
 ```bash
-# curl -L "https://github.com/grafana/loki/releases/download/${CIRCLE_TAG}/loki-linux-amd64.gz
-
-# extract the binary
-$ gunzip "loki-linux-amd64.gz"
-
-# make sure it is executable
-$ chmod a+x "loki-linux-amd64"
-```
-
+$ wget $YOUR_BINARY
```

Comment:
Why not just update previous instructions to use zip instead (without writing to /usr/local/bin specifically)? Furthermore, curl is everywhere, wget is not. 

```suggestion
$ curl -O -L "https://github.com/grafana/loki/releases/download/${CIRCLE_TAG}/loki-linux-amd64.zip"

# extract the binary
$ unzip "loki-linux-amd64.zip"

 # make sure it is executable
 $ chmod a+x "loki-linux-amd64"
```

Reply:
done.